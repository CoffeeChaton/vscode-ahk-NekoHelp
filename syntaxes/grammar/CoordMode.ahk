CoordMode, ToolTip, Screen
CoordMode, Pixel, Screen
CoordMode, Mouse, Screen
CoordMode, Caret, Screen
CoordMode, Menu, Screen

CoordMode, ToolTip, Screen
CoordMode, ToolTip, Relative
CoordMode, ToolTip, Window
CoordMode, ToolTip, Client


CoordMode, ToolTip
CoordMode, Pixel
CoordMode, Mouse
CoordMode, Caret
CoordMode, Menu
