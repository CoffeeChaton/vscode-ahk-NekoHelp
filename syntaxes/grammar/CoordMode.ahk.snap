>CoordMode, ToolTip, Screen
#^^^^^^^^^ source.ahk meta.command.line.coordmode.ahk support.function.command.coordmode.ahk
#         ^^ source.ahk meta.command.line.coordmode.ahk
#           ^^^^^^^ source.ahk meta.command.line.coordmode.ahk keyword.command.option.param.coordmode.ahk
#                  ^^ source.ahk meta.command.line.coordmode.ahk
#                    ^^^^^^ source.ahk meta.command.line.coordmode.ahk keyword.command.option.param.coordmode.ahk
>CoordMode, Pixel, Screen
#^^^^^^^^^ source.ahk meta.command.line.coordmode.ahk support.function.command.coordmode.ahk
#         ^^ source.ahk meta.command.line.coordmode.ahk
#           ^^^^^ source.ahk meta.command.line.coordmode.ahk keyword.command.option.param.coordmode.ahk
#                ^^ source.ahk meta.command.line.coordmode.ahk
#                  ^^^^^^ source.ahk meta.command.line.coordmode.ahk keyword.command.option.param.coordmode.ahk
>CoordMode, Mouse, Screen
#^^^^^^^^^ source.ahk meta.command.line.coordmode.ahk support.function.command.coordmode.ahk
#         ^^ source.ahk meta.command.line.coordmode.ahk
#           ^^^^^ source.ahk meta.command.line.coordmode.ahk keyword.command.option.param.coordmode.ahk
#                ^^ source.ahk meta.command.line.coordmode.ahk
#                  ^^^^^^ source.ahk meta.command.line.coordmode.ahk keyword.command.option.param.coordmode.ahk
>CoordMode, Caret, Screen
#^^^^^^^^^ source.ahk meta.command.line.coordmode.ahk support.function.command.coordmode.ahk
#         ^^ source.ahk meta.command.line.coordmode.ahk
#           ^^^^^ source.ahk meta.command.line.coordmode.ahk keyword.command.option.param.coordmode.ahk
#                ^^ source.ahk meta.command.line.coordmode.ahk
#                  ^^^^^^ source.ahk meta.command.line.coordmode.ahk keyword.command.option.param.coordmode.ahk
>CoordMode, Menu, Screen
#^^^^^^^^^ source.ahk meta.command.line.coordmode.ahk support.function.command.coordmode.ahk
#         ^^ source.ahk meta.command.line.coordmode.ahk
#           ^^^^ source.ahk meta.command.line.coordmode.ahk keyword.command.option.param.coordmode.ahk
#               ^^ source.ahk meta.command.line.coordmode.ahk
#                 ^^^^^^ source.ahk meta.command.line.coordmode.ahk keyword.command.option.param.coordmode.ahk
>
>CoordMode, ToolTip, Screen
#^^^^^^^^^ source.ahk meta.command.line.coordmode.ahk support.function.command.coordmode.ahk
#         ^^ source.ahk meta.command.line.coordmode.ahk
#           ^^^^^^^ source.ahk meta.command.line.coordmode.ahk keyword.command.option.param.coordmode.ahk
#                  ^^ source.ahk meta.command.line.coordmode.ahk
#                    ^^^^^^ source.ahk meta.command.line.coordmode.ahk keyword.command.option.param.coordmode.ahk
>CoordMode, ToolTip, Relative
#^^^^^^^^^ source.ahk meta.command.line.coordmode.ahk support.function.command.coordmode.ahk
#         ^^ source.ahk meta.command.line.coordmode.ahk
#           ^^^^^^^ source.ahk meta.command.line.coordmode.ahk keyword.command.option.param.coordmode.ahk
#                  ^^ source.ahk meta.command.line.coordmode.ahk
#                    ^^^^^^^^ source.ahk meta.command.line.coordmode.ahk keyword.command.option.param.coordmode.ahk
>CoordMode, ToolTip, Window
#^^^^^^^^^ source.ahk meta.command.line.coordmode.ahk support.function.command.coordmode.ahk
#         ^^ source.ahk meta.command.line.coordmode.ahk
#           ^^^^^^^ source.ahk meta.command.line.coordmode.ahk keyword.command.option.param.coordmode.ahk
#                  ^^ source.ahk meta.command.line.coordmode.ahk
#                    ^^^^^^ source.ahk meta.command.line.coordmode.ahk keyword.command.option.param.coordmode.ahk
>CoordMode, ToolTip, Client
#^^^^^^^^^ source.ahk meta.command.line.coordmode.ahk support.function.command.coordmode.ahk
#         ^^ source.ahk meta.command.line.coordmode.ahk
#           ^^^^^^^ source.ahk meta.command.line.coordmode.ahk keyword.command.option.param.coordmode.ahk
#                  ^^ source.ahk meta.command.line.coordmode.ahk
#                    ^^^^^^ source.ahk meta.command.line.coordmode.ahk keyword.command.option.param.coordmode.ahk
>
>
>CoordMode, ToolTip
#^^^^^^^^^ source.ahk meta.command.line.coordmode.ahk support.function.command.coordmode.ahk
#         ^^ source.ahk meta.command.line.coordmode.ahk
#           ^^^^^^^ source.ahk meta.command.line.coordmode.ahk keyword.command.option.param.coordmode.ahk
>CoordMode, Pixel
#^^^^^^^^^ source.ahk meta.command.line.coordmode.ahk support.function.command.coordmode.ahk
#         ^^ source.ahk meta.command.line.coordmode.ahk
#           ^^^^^ source.ahk meta.command.line.coordmode.ahk keyword.command.option.param.coordmode.ahk
>CoordMode, Mouse
#^^^^^^^^^ source.ahk meta.command.line.coordmode.ahk support.function.command.coordmode.ahk
#         ^^ source.ahk meta.command.line.coordmode.ahk
#           ^^^^^ source.ahk meta.command.line.coordmode.ahk keyword.command.option.param.coordmode.ahk
>CoordMode, Caret
#^^^^^^^^^ source.ahk meta.command.line.coordmode.ahk support.function.command.coordmode.ahk
#         ^^ source.ahk meta.command.line.coordmode.ahk
#           ^^^^^ source.ahk meta.command.line.coordmode.ahk keyword.command.option.param.coordmode.ahk
>CoordMode, Menu
#^^^^^^^^^ source.ahk meta.command.line.coordmode.ahk support.function.command.coordmode.ahk
#         ^^ source.ahk meta.command.line.coordmode.ahk
#           ^^^^ source.ahk meta.command.line.coordmode.ahk keyword.command.option.param.coordmode.ahk
>